geniza/entities/templates/admin/entities/place/change_form.html

```diff
@@ -0,0 +1,10 @@
+{% extends "admin/change_form.html" %}
+
+{# Render mixed normal and inline fieldsets #}
+{% block field_sets %}
+    {% include "admin/corpus/document/snippets/mixed_inlines_fieldsets.html" %}
```

Comment:
should this be set at a higher level in a common template if it's used across multiple admin models?